__test__/groups.test.ts

```diff
@@ -2,7 +2,6 @@ import { Logger } from "gasmask";
 global.Logger = Logger;
 
 import { getRandomlyGeneratedGroupsMap, getRandomlyGeneratedGroupsTable, getRandomlyGeneratedSupergroupsTable, Mock } from "./testUtils";
-import { getMembersFromGroups } from "../src/main/groups";
```

Comment:
Importing the function to test at the top of the file means that any mocking does not apply to this version of the function that was imported and is eventually called. This happens because JS is interpreted and the file is parsed top to bottom meaning if we are doing any mocking, it must be setup first and then the function/module being tested has to be imported with the mocks that were setup